packages/core/src/lavadome.mjs

```diff
@@ -29,58 +29,62 @@ navigation.addEventListener('navigate', event => {
     }
 });
 
-export function LavaDome(host, opts) {
-    opts = options(opts);
-    
-    // make exported API tamper-proof
-    defineProperties(this, {text: {value: text}});
-
-    // get/create shadow for host (empty shadow content if there's any already)
-    const shadow = getShadow(host, opts);
-    replaceChildren(shadow);
+function defineCopy(instance, input) {
+    return async function copy() {
+        const type = 'text/plain';
```

Comment:
how can we support other mimetypes?

Reply:
The whole project currently focuses on plain text only because of how hard it is to safely present more complex types.

Generally, this is a question for another day, but i might be able to answer your question if you have an example in mind for "other mimetypes".

Reply:
a PNG.  say it's a QR code for initializing TOTP that we don't want to expose

Reply:
Very interesting, perhaps sometime far in the future